src/core/drive/head_snapshot.ts

```diff
@@ -55,57 +65,57 @@ export class HeadSnapshot extends Snapshot {
       } else {
         return result
       }
-    }, [])
+    }, [] as Element[])
   }
 
-  getMetaValue(name) {
+  getMetaValue(name: string): string | null {
     const element = this.findMetaElementByName(name)
     return element ? element.getAttribute("content") : null
   }
 
-  findMetaElementByName(name) {
+  findMetaElementByName(name: string) {
     return Object.keys(this.detailsByOuterHTML).reduce((result, outerHTML) => {
       const {
-        elements: [element]
+        elements: [element],
       } = this.detailsByOuterHTML[outerHTML]
       return elementIsMetaElementWithName(element, name) ? element : result
-    }, undefined | undefined)
+    }, undefined as Element | undefined)
```

Comment:
Not sure why you fixed this bug. We were keeping it on purpose so we could fix it later and then pretend like we did some actual work. Change it to `(undefined | undefined) as Element | undefined` to preserve it.